### ISSUE TITLE
Upgrade the code

### ISSUE THREAD

**Issue by faierbol** [marketplacekit/marketplacekit#140]
Hi all, does anybody tried to upgrade this code to laravel 7 or 8?

**Comment by jamesdigid** [comment]
Yes, i made an attempt but ran into several issues with incompat dependencies perhaps i'll take another stab at it if successful i'll make a PR

**Comment by ugintl** [comment]
+1 

I think the authors are not interested anymore. Such a good project, but I guess they have their own problems.
